docker/dockerfile.tri

```diff
@@ -208,8 +208,8 @@ ENV CPATH=/usr/local/include:$CPATH
 
 # Install HugeCTR
 RUN apt update -y && apt install rapidjson-dev -y
-RUN git clone https://github.com/NVIDIA-Merlin/HugeCTR.git /repos/HugeCTR && \
-      cd /repos/HugeCTR && if [ "$RELEASE" == "true" ] && [ ${HUGECTR_VER} != "vnightly" ]; then git fetch --all --tags && git checkout tags/${HUGECTR_VER}; else git checkout master; fi && \
+RUN git clone https://gitlab-master.nvidia.com/dl/hugectr/hugectr.git /repos/HugeCTR && \
```

Comment:
We can't use the private Gitlab version here - can we get hugectr to update the GitHub version?

Reply:
Yes @benfred I am using this temporary to test a new feature.